azure.yaml

```diff
@@ -1,17 +1,19 @@
-name: chat-rag-openai-csharp-prompty
+name: contoso-chat-csharp-prompty
 metadata:
-  template: chat-rag-openai-csharp-prompty@0.0.1
+  template: contoso-chat-csharp-prompty@0.0.1
+services:
+  api:
+    host: containerapp
+    language: csharp
+    project: src/ContosoChatAPI/ContosoChatAPI
 hooks:
   postprovision:
     posix:
       shell: sh
       continueOnError: false
       interactive: true
-      run: infra/hooks/postprovision.sh
```

Comment:
Each hook configuration needs a `run` parm.  You should likely put this line back in.